libloki/api.js

```diff
@@ -1,47 +1,128 @@
-/* global window, textsecure, dcodeIO, StringView, ConversationController */
+/* global window, textsecure, dcodeIO, StringView, ConversationController, _ */
+/* eslint-disable no-bitwise */
 
 // eslint-disable-next-line func-names
 (function() {
   window.libloki = window.libloki || {};
 
-  async function sendBackgroundMessage(pubKey) {
-    return sendOnlineBroadcastMessage(pubKey);
+  const DebugFlagsEnum = {
+    GROUP_SYNC_MESSAGES: 1,
+    CONTACT_SYNC_MESSAGES: 2,
+    AUTO_FRIEND_REQUEST_MESSAGES: 4,
+    SESSION_REQUEST_MESSAGES: 8,
+    SESSION_MESSAGE_SENDING: 16,
+    SESSION_BACKGROUND_MESSAGE: 32,
+    GROUP_REQUEST_INFO: 64,
+    NORMAL_FRIEND_REQUEST_MESSAGES: 128,
+    // If you add anything, be sure it is bitwise safe! (unique and 2 multiples)
+  };
+
+  const debugFlags =
+    DebugFlagsEnum.GROUP_SYNC_MESSAGES |
+    DebugFlagsEnum.CONTACT_SYNC_MESSAGES |
+    DebugFlagsEnum.AUTO_FRIEND_REQUEST_MESSAGES |
+    DebugFlagsEnum.SESSION_REQUEST_MESSAGES |
+    DebugFlagsEnum.SESSION_MESSAGE_SENDING |
+    DebugFlagsEnum.SESSION_BACKGROUND_MESSAGE |
+    DebugFlagsEnum.NORMAL_FRIEND_REQUEST_MESSAGES |
+    DebugFlagsEnum.GROUP_REQUEST_INFO;
```

Comment:
this  is not ideal. Do you have a better idea?

Reply:
You can just add a new Flag `DebugFlagsEnum.ALL` which has the value `255` (value of highest flag - 1)

Reply:
Good idea